src/components/Courses.jsx

```diff
@@ -0,0 +1,77 @@
+
+import {useState, useEffect} from 'react';
+import {Card, Row, Container, Accordion} from 'react-bootstrap';
+
+const Course = () => {
+    let [courses, setCourses] = useState(null);
+    let [loading, setLoading] = useState(true);
+
+    useState(()=>{
+        setLoading(true);
+        fetch('url')
+        .then(response => response.json())
+        .then(data =>{
+            setLoading(false);
+            setCourses(data);
+        })
+    }, []);
+
+    if(!loading){
+        if(courses != null){
+            return(
+                <>
```

Comment:
Do you need these Fragments? Seems like you can just have `<Container>` be the parent